### PR TITLE
fix(chat): home page add delete button to custom app, remove short description from card view (Issue #2279, #2334)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Add Delete button on application view for custom app on Home page
On marketplace on card view of All Applications and My Applications should be visible only short description

Issues:

- Issue #2279 
- Issue #2334 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by ai-dial-actions** [comment]
Hey there and thank you for opening this pull request! 👋🏼

We require pull request titles to follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) and it looks like your proposed title needs to be adjusted.

Details:

```
Unknown release type "ix" found in pull request title "ix(chat): home page add delete button to custom app, remove short description from card view (Issue #2279, #2334)". 

Available types:
 - fix
 - hotfix
 - feat
 - feature
 - chore
```

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2391.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — CHANGES_REQUESTED** [review]
Divider for short description is not "\n"

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2391.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const ApplicationDetailsFooter = ({
             className="shrink-0 text-accent-primary md:hidden [&_path]:fill-current"
             size={24}
           /> */}
-          {isMyAppsTab && (
+          {(isMyAppsTab || (!isMyAppsTab && isMyApp)) && (
```
Comment (IlyaBondar) [inline apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx]:
```suggestion
          {(isMyAppsTab || isMyApp) && (
```

File: apps/chat/src/utils/app/application.ts [hunk 2]
```diff
@@ -120,12 +121,22 @@ export const isQuickApp = (entity: DialAIEntityModel) => {
...
+  return isShort
+    ? description.split(DESCRIPTION_DELIMITER_REGEX)[0]
+    : description;
 };
 
+export const getModelDescription = (entity: DialAIEntityModel) =>
+  getDescription(entity);
+
+export const getModelShortDescription = (entity: DialAIEntityModel) =>
+  getDescription(entity, true);
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/application.ts]:
```suggestion
export const getModelDescription = (entity: DialAIEntityModel) => {
  return entity.description
    ? entity.description.split(QUICK_APP_CONFIG_DIVIDER)[0]
    : '';

export const getModelShortDescription = (entity: DialAIEntityModel) =>
  getModelDescription(entity).split(DESCRIPTION_DELIMITER_REGEX)[0];
```

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2391.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2391.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**